egs/mobvoi/v1/run_e2e.sh

```diff
@@ -0,0 +1,294 @@
+#!/bin/bash
+
+stage=0
+
+
+. ./cmd.sh
+. ./path.sh
+. utils/parse_options.sh
+
+set -euo pipefail
+
+if [ $stage -le 0 ]; then
+  local/mobvoi_data_download.sh
+  echo "$0: Extracted all datasets into data/download/"
+fi
+
+if [ $stage -le 1 ]; then
+  echo "$0: Splitting datasets..."
+  local/split_datasets.sh
+  echo "$0: text and utt2spk have been generated in data/{train|dev|eval}."
+fi
+    
+if [ $stage -le 2 ]; then
+  echo "$0: Preparing wav.scp..."
+  local/prepare_wav.py data
+  echo "wav.scp has been generated in data/{train|dev|eval}."
+fi
+
+if [ $stage -le 3 ]; then
+  echo "$0: Extracting MFCC..."
+  for folder in train dev eval; do
+    dir=data/$folder
+    utils/fix_data_dir.sh $dir
+    steps/make_mfcc.sh --cmd "$train_cmd" --nj 16 $dir
+    steps/compute_cmvn_stats.sh $dir
+    utils/fix_data_dir.sh $dir
+    utils/data/get_utt2dur.sh $dir
+    utils/validate_data_dir.sh $dir
+  done
+fi
+
+if [ $stage -le 4 ]; then
+  echo "$0: Post processing transcripts..."
+  for folder in train dev eval; do
+    dir=data/$folder
+    cat $dir/text | awk '{if ($2=="嗨小问" || $2=="嗨小问嗨小问") {print $1,"嗨小问";} else {print $1,"FREETEXT"}}' > $dir/text.tmp || exit 1
+    cat $dir/text.tmp > $dir/text || exit 1
+    rm -f $dir/text.tmp 2>/dev/null || true
+  done
+fi
+
+if [ $stage -le 5 ]; then
+  echo "$0: Preparing dictionary and lang..."
+  local/prepare_dict.sh
+  utils/prepare_lang.sh --num-sil-states 1 --num-nonsil-states 4 --sil-prob 0.5 \
+    --position-dependent-phones false \
+    data/local/dict "<sil>" data/lang/temp data/lang
+fi
+
+if [ $stage -le 6 ]; then
+  id_sil=`cat data/lang/words.txt | grep "<sil>" | awk '{print $2}'`
+  id_freetext=`cat data/lang/words.txt | grep "FREETEXT" | awk '{print $2}'`
+  id_word=`cat data/lang/words.txt | grep "嗨小问" | awk '{print $2}'`
```

Comment:
```sh
export LC_ALL=en_US.UTF-8
id_word=`cat data/lang/words.txt | grep "嗨小问" | awk '{print $2}'`
export LC_ALL=C
```

----
otherwise, id_word=`cat data/lang/words.txt | grep "嗨小问" | awk '{print $2}'`
returns `file`

Reply:
@csukuangfj thanks for helping with the review of this!